llvm/test/MC/Hexagon/directive-attribute-err.s

```diff
@@ -0,0 +1,24 @@
+/// attribute parsing error cases.
+
+// RUN: not llvm-mc -triple=hexagon -filetype=asm %s 2>&1 \
+// RUN:   | FileCheck %s
+
+  .attribute Tag_unknown_name, 0
+// CHECK: error: attribute name not recognized: Tag_unknown_name
```

Comment:
Newer MC tests try to test the error line/column information like: `[[#@LINE-1]]:4: error:`

Reply:
Done